Cython/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Any
+
+from .Shadow import __version__
+
+from .Shadow import *
+
+
+def load_ipython_extension(ip: Any) -> None: ...
```

Comment:
`ip` should have [InteractiveShell](https://ipython.readthedocs.io/en/stable/api/generated/IPython.core.interactiveshell.html#IPython.core.interactiveshell.InteractiveShell) type

Reply:
like [here](https://ipython.readthedocs.io/en/stable/config/extensions/index.html#writing-extensions)

Reply:
Not everyone has `IPython` installed so we can't import that.

Besides, this will simply be called by `IPython`, so it is not that important that the user gets these typings.

The only reason I defined this in the first place was to make MyPy happy and not cause any weird issues in the future with false assumptions.

Reply:
It might be worth adding this information there as a comment

Reply:
```suggestion
# Internal interface for IPython. Not further typed since it's not meant for end users.
def load_ipython_extension(ip: Any) -> None: ...
```